tomb-wasm/src/metadata/crypto.rs

```diff
@@ -4,6 +4,7 @@ use wasm_bindgen::{prelude::wasm_bindgen, JsValue};
 use crate::fetch::http::get_data;
 
 #[wasm_bindgen]
+#[derive(Debug)]
```

Comment:
We probably don't want private keys to allow any form of printable output (even Debug). It's just a safety precaution against these values showing up in any logs or diagnostic output. If this was added because it a type embeds it and is also implementing Debug, you'll probably need to create a custom Debug implementation to omit the field but that's really not too bad.